frontend/src/main/java/olumofe/raphael/frontend/CarController.java

```diff
@@ -0,0 +1,57 @@
+package olumofe.raphael.frontend;
+
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.*;
+import starter.project.domain.Car;
+
+import java.util.*;
+
+@RestController
+@Component
+public class CarController {
```

Comment:
where are the other methods? the PUT ones?

Reply:
After we resolved the conflicts yesterday they disappeared.

Reply:
You should be able to pick and choose what to keep and delete when you resolve conflicts. What did you merge?